backend/qemu.pm

```diff
@@ -271,20 +271,14 @@ sub load_snapshot {
     return $rsp;
 }
 
-sub runcmd {
-    diag "running " . join(' ', @_);
-    local $SIG{CHLD} = 'IGNORE';
-    return CORE::system(@_);
```

Comment:
You are reverting https://github.com/os-autoinst/os-autoinst/pull/534 basically - so I object

Reply:
I will take `system` from CORE package back and avoid autodie